pom.xml

```diff
@@ -153,7 +154,8 @@ Import-Package: \\
   ${bnd.exportpackage}
 -sources: false
 -contract: *
--includeresource: ${bnd.includeresource}]]></bnd>
+-includeresource: ${bnd.includeresource}
+-fixupmessages: ${bnd.fixupmessages}]]></bnd>
```

Comment:
What do these changes have to do with this PR?

Reply:
The `-fixupmessages` is required for embedding the jython JAR. Without this config Bnd errors. By adding it this way future add-ons can also use it. The only other way I see to customize the bnd instructions would be to duplicate the whole plugin configuration in each add-on POM.

If possible I would prefer to start using `bnd.bnd` files similarly like we do in the core and osgiify. That would provide an easier and more standard way to customize the bnd instructions per add-on. It might also fix the issue of always having to add all those optional imports for embedded JARs. You can see that in https://github.com/openhab/openhab-addons/pull/7208 adding all those optional imports wasn't necessary for the embedded JAR.